lib/player.rb

```diff
@@ -0,0 +1,20 @@
+class Player
+
+  attr_reader :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def rock
```

Comment:
We've got these defined in Choice as well. Could this be something we could extract?